app/models/attend.rb

```diff
@@ -0,0 +1,8 @@
+class Attend < ApplicationRecord
```

Comment:
there is a future-ticket for that

Right now it's enough to just have a static page, that we can route to, for people who want to attend